NitroxClient/GameLogic/DevConsoleSpawnEvent.cs

```diff
@@ -0,0 +1,58 @@
+﻿using System;
+using System.Collections.Generic;
+using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.DataStructures;
+using NitroxModel.DataStructures.GameLogic;
+using NitroxModel.Logger;
+using NitroxModel.Packets;
+using NitroxModel_Subnautica.Helper;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic
+{
+    public class DevConsoleSpawnEvent
```

Comment:
There is a DevConsole.cs (which has an incorrect class name inside called NitroxConsole).  Should we rename the incorrect class and move this logic there?